drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -1177,9 +1181,33 @@ static int nxp_wifi_status(const struct device *dev, struct wifi_iface_status *s
 }
 
 #if defined(CONFIG_NET_STATISTICS_WIFI)
+#ifdef CONFIG_NXP_WIFI_GET_LOG
+static int nxp_wifi_get_detail_stats(int bss_type, wlan_pkt_stats_t *stats)
+{
+	int ret = -1;
```

Comment:
If you set here `int ret = -ENODEV;` then the last else branch can be omitted. Now setting the ret to -1 is useless as the else branch sets the value anyway.